src/SampleWebApp/Controllers/AuthorssApiController.cs

```diff
@@ -0,0 +1,149 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Data;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Controllers
+{
+    [Produces("application/json")]
```

Comment:
This seems to be from the pre RC1 days. 
The only thing I could find out is that this was a way for content negotiation. 
Am not sure if this is needed or not. 

@rynowak do you know who would be able to help out with this?